app/components/visualizer/main-viz/templates/chart.jsx

```diff
@@ -0,0 +1,48 @@
+import D3Chart from "./D3Chart.js"
+
+class Chart extends React.Component {
+  componentDidMount() {
+    let el = React.findDOMNode(this);
+    this.chart = new D3Chart(el, {
+      width: "100%",
```

Comment:
can you remove my commits from this PR? 

`git rebase --interactive master` then just delete my commits